tests/test-cases.js

```diff
@@ -1585,11 +1585,11 @@ var PARSE_TESTS = [
 		}
 	},
 	{
-		description: "Skip First N number of lines , with header and 3 rows",
```

Comment:
Please do not remove existing test suite but add a new test latter.

Reply:
I did'nt remove the test, I corrected it. 
As I already mentioned, the first line should be expected to be the header row in the result. (because it should be skipped with skipFirstNLines=1